packages/rocketchat-lib/server/models/Rooms.js

```diff
@@ -5,9 +5,16 @@ class ModelRooms extends RocketChat.models._Base {
 	constructor(...args) {
 		super(...args);
 
-		this.tryEnsureIndex({ name: 1 }, { unique: 1, sparse: 1 });
+
 		this.tryEnsureIndex({ default: 1 });
 		this.tryEnsureIndex({ t: 1 });
+
+		// ttrc additions or edits from Marc 
```

Comment:
Nonde of these indexes are necessary any more. Mine from line 13 was required as the unique index on "name" existed and was generating conflict on having multiple channels with the same name.

As for the "usernames" index, it should be removed too

Reply:
Removed both lines and submitted a commit